src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
-export interface Figure {
 
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
would be better to make Shape a enum

Reply:
I think both would benefit from enum. In real project there might be much more shapes and colors. I just made it type for simplicity.